integration_tests/features/WalletTransactions.feature

```diff
@@ -313,3 +313,21 @@ Feature: Wallet Transactions
     And I wait 5 seconds
     Then I restart wallet WALLET_RECV
     Then I wait for wallet WALLET_RECV to have at least 1000000 uT
+
+@critical
+  Scenario: Wallet should cancel stale transactions
```

Comment:
```suggestion
  Scenario: Wallet should manually cancel stale transactions
```

Reply:
The focus must be to do this manually and should not happen automatically; this is not clear reading the name of the test